### ISSUE TITLE
Spec: Scroll Triggered Animations #272, Additional considerations when drafting accessibility considerations section

### ISSUE THREAD

**Issue by ehind67** [w3c/a11y-tracking#286]
## Proposed comment
@matatk has create an comment (https://github.com/w3ctag/design-reviews/issues/1167#issuecomment-3667157396) related to high level accessibility consideration section.  COGA would additional ask that this section include notes related to the following

- make it clear that involuntarily encountering animation can cause significant harm to users with vestibular or binocular-vision disorders. 
- Is there a place for alternative text for people who do not want animation? 
- Is there a still picture for those who can see but need to avoid motion and  animation. 
Motion can cause:
- Dizziness
- Startle response 
- Sensory overload from lights and sounds, followed by physical symptoms.
- Anxiety 
- Cybersickness (new term; [this source](https://balanceanddizziness.org/disorders/vestibular-disorders/motion-and-cyber-sickness/#:~:text=Cyber%20sickness%20is%20a%20subset,up%20by%20robust%20scientific%20evidence.) hasn’t been vetted by COGA yet)
-

Section_etc_title
none - this is part of an ask for an accessibility consideration section to be added

> quote_the_text
none

## Instructions

This follows the process at https://www.w3.org/WAI/APA/wiki/Horizontal_Review#For_Specifications_2

1. Create the review comment you want to propose by replacing the prompts above these instructions, but **LEAVE ALL THE INSTRUCTIONS INTACT**.

2. Set a label to identify the spec: this starts with s: followed by the spec's short name. If you are unable to do that, ask a W3C staff contact to help.

3. Ask the APA WG to review your comment.

4. After discussion with the APA WG, raise an issue in the repository of the WG that owns the spec. Use the text above these instructions as the starting point for that comment, but add any suggestions that arose from the APA WG. In the other WG's repo, add an 'a11y-needs-resolution' label to the new issue.

5. Delete the text below that says 'url_for_the_issue_raised', then add in its place the URL for the issue you raised in the other WG's repository. Do NOT remove the initial '§ '. Do NOT use \[...](...) notation – you need to delete the placeholder, then paste the URL.

6. Remove the 'pending' label, and add a 'needs-resolution' tag to this tracker issue. 

7. Edit this issue to **REMOVE ALL THE INSTRUCTIONS & THE PROPOSED COMMENT**, ie. the line below that is '---' and all the text before it to the very start of the issue.

---

**This is a tracker issue.** Only discuss things here if they are a11y group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ url_for_the_issue_raised

**Comment by matatk** [comment]
Per [discussion last week](https://www.w3.org/2026/01/14-apa-minutes.html#5947) it looks like this is a general accessibility consideration, and I don't see an 'accessibility considerations' section in the full [CSS Animations spec](https://www.w3.org/TR/css-animations-2/) so I wonder if we should enquire with the CSS WG on if there _is_ such a section, and propose this as the basis of it.

First we should probably check how much of this is covered in WCAG, as WCAG is very well-known amongst CSS WG, and generally well-known amongst developers - but if there are _specific_ considerations about how to use the animation features of CSS, they would be good to include in this doc.
